src/css/reset.css

```diff
@@ -0,0 +1,150 @@
+/*
+Project specific resets go here
+*/
+* {
+    box-sizing: border-box;
+  }
+  a,
+  button, .toggle {
+    cursor: pointer;
+  }
+  /* removes blue highlight on Chrome 8 */
+  *:focus {
+    outline: none;
+  }
+
+  /* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+  
+  html,
```

Comment:
is it possible to import this from an npm package instead of creating a copy here?